docker-compose.ac.yml

```diff
@@ -54,5 +54,29 @@ services:
     logging:
       options: { max-file: "5", max-size: 100m }
     user: "2000:2000"
-
+  ms-ledger:
+    image: "automated-checkout/ms-ledger:dev"
+    container_name: automated-checkout_ms-ledger
+    depends_on:
+      data:
+        condition: service_started
+      command:
+        condition: service_started
+    environment:
+      EDGEX_SECURITY_SECRET_STORE: "false"
+      REGISTRY_HOST: edgex-core-consul
+      SERVICE_HOST: ms-ledger
+      APPCUSTOM_INVENTORYENDPOINT: "http://ms-inventory:48095/inventory"
+    hostname: ms-ledger
+    networks:
+      edgex-network: {}
+    ports:
+    - 48093:48093/tcp
```

Comment:
Should inventory also restrict to 127.0.0.1 access?

Reply:
I see original has 127.0.0.1 restriction, I will honor that.